tempesta_fw/t/functional/cache/test_purge.py

```diff
@@ -60,10 +59,9 @@ def proxy_chain(method, uri):
     return chain
 
 def purge_chain(uri):
-    chain = functional.base_message_chain(uri=uri, method='PURGE')
-    chain.no_forward()
+    chain = functional.base_message_chain(uri=uri, method='PURGE', forward=False)
     headers = [
         'Connection: keep-alive',
         'Content-Length: 0']
-    chain.response = deproxy.Response.create(200, headers, date=True, body='')
+    chain.response = deproxy.Response.create(200, headers, date=chain.response.headers["Date"], body='')
```

Comment:
It is already there, I just forgot to drop the obsolete helpers from this testcase.